subprojects/gradle/buildchecks/src/main/kotlin/com/avito/android/plugin/build_param_check/UniqueRClassesTask.kt

```diff
@@ -13,11 +13,16 @@ abstract class UniqueRClassesTask : DefaultTask() {
     fun check() {
         val apps = project.subprojects.filter { it.isAndroidApp() }
 
-        apps.forEach { app ->
+        apps.stream().parallel().forEach { app ->
```

Comment:
Did you check that `forEech` gave increasion?

Reply:
The whole task works longer becuase of additional configurations to check: 1 min become 3 min.
Avito app is on critical path, other applications much faster.

Without parallelization the task tooks 6 min.